### DISCUSSION TITLE
p:menuBar - div with tabindex 0 at the beginning of the menu entries is read as empty by screen readers.

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#231] [PrimeFaces] [answered]
p:menuBar rendes an empty div with tabindex 0 at the beginning of the menu entries, which is read out as empty by screen readers (NVDA).

I would expect the screen reader to read me the name of the active menu item. Is this possible?

The behavior can be recreated in your show case for the MenuBar element.

**Comment by Bukama** [comment]
What @hgfrey means that the screenreader really reads out "empty" every time and with a non negative tab index this empty div is also focusable by keyboard users.

**Reply by melloware** [reply, marked as answer]
Fixed in PrimeFaces 14.0.1!

~It's this open issue.~ Read that whole thread: https://github.com/primefaces/primefaces/issues/6561

**Comment by melloware** [comment]
Fixed in 14.0.1 which is in Maven Central now.
